content/v2.0/query-data/guides/monitor-states.md

```diff
@@ -0,0 +1,192 @@
+          )
+          ```
+
+2. Run `stateCount()` to search each point for the specified value:
+
+    - For the first point that evaluates `true`, the state count is set to `1`. For each consecutive point that evaluates `true`, the state count increases by 1.
+    - If the state is `false`, the state count is reset to `-1`.
+
+### Example query with stateCount()
+
+The following query searches the `doors` bucket over the past 5 minutes to find how many points have been counted where `_value` is `closed`.
+
+```js
+  from(bucket: "doors")
+  |> range(start: -5m)
+  |> stateDuration(fn: (r) =>
+   r._value == "closed",
+   column: "door_closed")
+```
+
+In this example, `door_closed` is the **State count** column. If you write data to the `doors` bucket every minute, the state count increases by `1` for each consecutive point where `_value` is `closed`. If `_value` is not `closed`, the state count is reset to `-1`.
+
+#### Query results
+
+Results for the example query above may look like this (for simplicity, we've omitted the measurement, tag, and field columns):
+
+```bash
+_time                   _value        door_closed
+2019-10-26T17:39:16Z    closed        1
+2019-10-26T17:40:16Z    closed        2
+2019-10-26T17:41:16Z    closed        3
+2019-10-26T17:42:16Z    open          -1
+2019-10-26T17:43:16Z    closed        1
+2019-10-26T17:44:27Z    closed        2
+```
+
+<!-- #### Example query to count machine state
+
+To check the machine state every minute (idle, assigned, or busy).
+
+```
+from(bucket: "servers")
+  |> range(start: -1h)
+  |> group(columns: ["r.machine_state"])
+(  |> filter(fn: (r) => r.machine_state == "idle" or r.machine_state == "assigned" or r.machine_state == "busy")) --does this help filter if there are more than 3 machine states?...or do next 3 lines do the same?
+  |> stateCount(fn: (r) => r.machine_state == "busy", column: "_count")
+  |> stateCount(fn: (r) => r.machine_state == "assigned", column: "_count")
+  |> stateCount(fn: (r) => r.machine_state == "idle", column: "_count")
+```
+
+In this query, InfluxDB searches the `servers` bucket over the past hour, counts records with a machine state of `idle`, `assigned` or `busy` and groups by the machine state:
+
+-->
+
+## Detect state changes
+
+Detect state changes with the `monitor.stateChanges()` function. To use the `monitor.stateChanges()` function, you must set up a **check** to query data (stored in the `_monitoring` bucket > `statuses` measurement > `_level` column; see [Monitor data and send alerts](v2.0/monitor-alert/) for more detail.
+
+1. In the InfluxDB user interface, click the **Monitoring and Alerting** icon from the sidebar.
+
+    {{< nav-icon "alerts" >}}
+
+2. If you haven't already, [create a check](v2.0/monitor-alert/checks/create/) that stores statuses (`CRIT`, `WARN`, `INFO`, `OK` or `ANY`) in the `_level` column. <!-- specify how to do this with monitor.check() function or in UI, with check threshold or deadman? -->
```

Comment:
~`[create a check](v2.0/monitor-alert/checks/create/)`~
`[create a check](/v2.0/monitor-alert/checks/create/)`

You're missing the leading slash in the link.